plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/lsp/model/aws/chat/FlareChatCommands.kt

```diff
@@ -64,3 +64,5 @@ const val MCP_SERVER_CLICK_REQUEST_METHOD = "aws/chat/mcpServerClick"
 const val OPEN_FILE_DIALOG_REQUEST_METHOD = "aws/chat/openFileDialog"
 const val LIST_RULES_REQUEST_METHOD = "aws/chat/listRules"
 const val RULE_CLICK_REQUEST_METHOD = "aws/chat/ruleClick"
+
+const val LIST_AVAILABLE_MODELS = "aws/chat/listAvailableModels"
```

Comment:
nit: can you please add this event in alphabetical order?

Reply:
I think it makes more sense to sort these by features, and also the other methods are not in alphabetical order either